notes.md

```diff
@@ -44,11 +44,6 @@ The default value of an Element is assumed when not present in the data stream.
 
 Digital Rights Management. See [Encryption]({{site.baseurl}}/notes.html#Encryption).
 
-# EBML Class
-
-A larger EBML Class typically means the Element has a lower probability/importance. A larger Class-ID can be used as a synch word in case the file is damaged. Elements that are used frequently, but do not need to act as a synch word, SHOULD have a small Class-ID.
```

Comment:
Is this point about length and sync in the ebml spec. It could be used in https://github.com/Matroska-Org/ebml-specification/blob/master/specification.markdown#element-id.